code/ecdsap256/Hacl.Impl.ECDSA.P256.Verification.Agile.fst

```diff
@@ -464,7 +558,7 @@ let ecdsa_verification_core alg publicKeyBuffer hashAsFelem r s mLen m xBuffer t
   r
 
 
-[@ (Comment "  This code is not side channel resistant")]
+(* [@ (Comment "  This code is not side channel resistant")] *)
```

Comment:
did the come become side-channel resistant or is there another issue related to C comments?